tpls/desul/include/desul/atomics/Thread_Fence_OpenMP.hpp

```diff
@@ -16,7 +16,8 @@ SPDX-License-Identifier: (BSD-3-Clause)
 namespace desul {
 namespace Impl {
 
-#if _OPENMP > 201800
+// FIXME_OPENMPTARGET_NVHPC: NVHPC compiler does not support memory-order-clause in the flush construct in OpenMP.
+#if _OPENMP > 201800 && !defined(_NVCOMPILER)
```

Comment:
This needs a pull request on the desul-side.

Reply:
Thanks. I put in a PR in desul for the same https://github.com/desul/desul/pull/107 .